src/views/TechnicalAssistance/AdviceLetterForm/NextSteps.tsx

```diff
@@ -0,0 +1,261 @@
+import React, { useCallback, useEffect } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { Trans, useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { ApolloError, useMutation } from '@apollo/client';
+import { yupResolver } from '@hookform/resolvers/yup';
+import {
+  ErrorMessage,
+  Fieldset,
+  Form,
+  FormGroup,
+  Radio,
+  TextInput
+} from '@trussworks/react-uswds';
+
+import HelpText from 'components/shared/HelpText';
+import Label from 'components/shared/Label';
+import TextAreaField from 'components/shared/TextAreaField';
+import { UpdateTrbAdviceLetterQuery } from 'queries/TrbAdviceLetterQueries';
+import {
+  UpdateTrbAdviceLetter,
+  UpdateTrbAdviceLetterVariables
+} from 'queries/types/UpdateTrbAdviceLetter';
+import {
+  AdviceLetterNextSteps,
+  StepComponentProps
+} from 'types/technicalAssistance';
+import { nextStepsSchema } from 'validations/trbRequestSchema';
+
+import { StepSubmit } from '../RequestForm';
+import Pager from '../RequestForm/Pager';
+
+const NextSteps = ({
+  trbRequestId,
+  adviceLetter,
+  setFormAlert,
+  setStepSubmit,
+  setIsStepSubmitting
+}: StepComponentProps) => {
+  const { t } = useTranslation('technicalAssistance');
+  const history = useHistory();
+
+  const { nextSteps, isFollowupRecommended, followupPoint } = adviceLetter;
+
+  const [update] = useMutation<
+    UpdateTrbAdviceLetter,
+    UpdateTrbAdviceLetterVariables
+  >(UpdateTrbAdviceLetterQuery);
+
+  const {
+    handleSubmit,
+    control,
+    watch,
+    formState: { isSubmitting, isDirty }
+  } = useForm<AdviceLetterNextSteps>({
+    resolver: yupResolver(nextStepsSchema),
+    defaultValues: {
+      nextSteps,
+      isFollowupRecommended,
+      followupPoint
+    }
+  });
+
+  /** Submit next steps fields and update advice letter */
+  const submit: StepSubmit = useCallback(
+    callback => {
+      /** Submits form and updates advice letter */
+      const submitForm = handleSubmit(
+        async formData => {
+          if (isDirty) {
+            // UpdateTrbAdviceLetter mutation
+            await update({
+              variables: {
+                input: {
+                  trbRequestId,
+                  ...formData,
+                  // If isFollowUpRecommended is set to false, clear followupPoint value
+                  followupPoint: formData.isFollowupRecommended
+                    ? formData.followupPoint
+                    : null
+                }
+              }
+            });
+          }
+        },
+        // Throw error to cause promise to fail
+        () => {
+          throw new Error('Invalid field submission');
+        }
+      );
+
+      // Submit form
+      return submitForm().then(
```

Comment:
Just a style comment - often time function/variable names are comments in themselves and should give information about what that function/variable does.  Comments are especially useful when that context is not always clear.  Comments are probably not needed when the naming of the function/variable do the job of conveying their purpose.